esp/src/eclwatch/nls/es/hpcc.js

```diff
@@ -704,41 +723,43 @@ define(
     title_BindingDefinition: "Definición de conexión",
     title_ClusterInfo: "Grupos",
     title_CodeGeneratorPermissions: "Permisos de generador de codigo",
+    title_DefinitionExplorer: "Explorador de Definiciónes",
+    title_Definitions: "Definiciónes",
     title_DESDL: "ESDL dinámico",
     title_DFUQuery: "Archivos Lógicos",
     title_DFUWUDetails: "Unidad de trabajo DFU",
     title_DirectoriesFor: "Directorios para",
-    title_Definitions: "Definiciónes",
-    title_DefinitionExplorer: "Explorador de Definiciónes",
     title_DiskUsage: "Utilización del disco",
     title_ECLPlayground: "ECL Playground",
     title_ErrorsWarnings: "Errores/Advertencias de",
     title_EventScheduleWorkunit: "Planificador de Eventos",
-    title_FilesPendingCopy: "Archivos pendientes para copiar",
     title_FileScopeDefaultPermissions: "Permisos de archive por defecto",
+    title_FilesPendingCopy: "Archivos pendientes para copiar",
     title_FoundFilesFor: "Archivos encontrados para",
     title_GetDFUWorkunits: "Unidades de trabajo DFU",
     title_Graph: "Gráficos",
     title_GraphPage: "título",
     title_Graphs: "Gráficos",
     title_GridDetails: "Cámbieme",
+    title_History: "Historia",
     title_HPCCPlatformECL: "ECL Watch - Hogar",
     title_HPCCPlatformFiles: "ECL Watch - Archivos",
     title_HPCCPlatformMain: "ECL Watch - Hogar",
     title_HPCCPlatformOps: "ECL Watch - Operaciones",
     title_HPCCPlatformRoxie: "ECL Watch - Roxie",
+    title_HPCCPlatformServicesPlugin: "ECL Watch - dispositivo opcional",
```

Comment:
what's the diff between line 750 and 751?

Reply:
@kunalaswani Seems like a duplicate.

Reply:
its a dup, removing the english version